api/src/client.rs

```diff
@@ -313,6 +340,54 @@ pub(crate) fn status_code_help<T: FnMut(&Response) -> String>(
     }
 }
 
+fn add_settings_url_to_context(
+    context: &str,
+    domain: Option<String>,
+    org_url_slug: &String,
+) -> String {
+    match domain {
+        Some(present_domain) => {
+            let settings_url = format!(
+                "https://{}/{}/settings",
+                present_domain.replace("api", "app"),
+                org_url_slug
+            );
+            format!(
+                "{}\nYour settings page can be found at: {}",
+                context, settings_url
+            )
+        }
+        None => String::from(context),
+    }
+}
+
+#[test]
+fn adds_settings_if_domain_present() {
+    let domain = url::Url::parse("https://api.fake-trunk.io/")
+        .ok()
+        .into_iter()
+        .flat_map(|url| {
+            url.domain()
+                .into_iter()
+                .map(String::from)
+                .collect::<Vec<String>>()
+        })
+        .next();
+    let final_context =
+        add_settings_url_to_context("base_context", domain, &String::from("fake-org-slug"));
+    assert_eq!(
+        final_context,
+        "base_context\nYour settings page can be found at: https://app.fake-trunk.io/fake-org-slug/settings",
```

Comment:
Nit: Maybe call out that this is a hint?
```suggestion
        "base_context\nHint - Your settings page can be found at: https://app.fake-trunk.io/fake-org-slug/settings",
```